summarization/summarizer_main.py

```diff
@@ -250,18 +261,24 @@ def _get_attribute_for_contents(self, content):
             if content.content_type == ContentType.IMAGE:
                 media = content
                 stamp_descriptor_embedding = media.img_description_embedding
+
+            elif content.content_type == ContentType.QUOTE:
+                quote = content
+                stamp_descriptor_embedding \
+                    = self._get_stamp_descriptor_for_embedded_content(content)
```

Comment:
Rename this method (self._get_stamp_descriptor_for_embedded_content) to accept quotes as well.

Also, why are we treating quotes as embedded content. We could also get proper sentence embeddings for quotes right?

Reply:
Ah yes. Thanks for the suggestion. Getting the proper embedding makes sense. Will update accordingly.